src/patternfly/layouts/Toolbar/styles.scss

```diff
@@ -2,16 +2,23 @@
 
 // The value of component scoped variables is always defined by a global variable:
 .pf-l-toolbar {
+  padding: 0 0 2em 2em;
+
   // Toolbar Section
-  --pf-l-toolbar__section--MarginTop: var(--pf-global--spacer--md);
+  --pf-l-toolbar__section--MarginTop: var(--pf-global--spacer--xl);
```

Comment:
changing the spacing around `.pf-l-toolbar__section` isn't part of this PR. you only want to change vertical space between `.pf-l-toolbar__item` and `.pf-l-toolbar__group` when those elements wrap and are stacked on top of one another.

Reply:
Thanks for the advice. Code settings has been readjusting in the latest build.